### DISCUSSION TITLE
can I use multiple buffers?

### DISCUSSION THREAD

**Discussion by ryoppippi** [olimorris/codecompanion.nvim#351] [Q&A] [answered]
can I use multiple buffers?

**Comment by olimorris** [comment]
You'll need to provide more context...more chat bufffers? Insert more buffers via the slash commands?!

**Comment by jedi04** [comment]
@olimorris How do i send currently open buffers as context to my chat ?


Thank you

**Reply by olimorris** [reply]
You can via the `/buffer` slash command. That will list all open buffers and you can select one to insert into the chat buffer.

**Reply by andtankian** [reply]
@olimorris
I was about to open a PR when I saw the instruction to open a discussion first. This one is the closest to my PR content.
Telescope provides a way to attach mappings to each available behavior in the launched picker and once the codecompanion provides the ability to select a buffer using telescope, it would be awesome to send to the chat buffer the content of multiple selected buffers at one single call to /buffer slash command.
Pressing Tab in the telescope picker will mark the current selected item list under the cursor, it would iterate over the content of each selected buffer and send to the chat buffer in a single shot.

What do you think about that?

![image](https://github.com/user-attachments/assets/f6eeb9f1-fd65-4dc3-addd-97b51b6e379b)

**Reply by olimorris** [reply]
@andtankian that would be awesome

**Reply by andtankian** [reply]
I'm gonna work on that then!

**Comment by acascais** [comment]
@andtankian how do you open that select buffer window?

**Comment by olimorris** [comment, marked as answer]
This has now been added to the plugin for Telescope and Mini.pick.
